packages/relay/src/lib/model.ts

```diff
@@ -20,6 +20,7 @@
 
 // Used for fake implementation of block history
 import {Status, TransactionRecord} from "@hashgraph/sdk";
+import {EthImpl} from "./eth";
```

Comment:
Hmm, seems weird to be importing this just for a formatting method.
Eventually (not a fix for now) I think we should move the converter functions out to the `formatter.ts` class